spec/metrician_spec.rb

```diff
@@ -412,5 +412,25 @@ def app
       end
 
     end
+
+    describe "rails" do
+      def app
+        Rails.application
```

Comment:
Re: https://github.com/Instrumental/metrician-ruby/pull/28/commits/b090ae7712ac0e6d630c1cf3347f0d09ded197e9#r131264357, I think it would be clearer if we could refer to this application by name `TestRailsApp`